pkg/operator/operator.go

```diff
@@ -19,8 +19,10 @@ import (
 	"encoding/base64"
 	"errors"
 	"fmt"
+	l "log"
```

Comment:
nit: usually we prefix with std if we're aliasing something from the std library
```suggestion
	stdlog "log"
```